platforms/platform-facebookmessenger/README.md

```diff
@@ -0,0 +1,143 @@
+---
+title: 'Facebook Messenger Platform Integration'
+excerpt: 'The Facebook Messenger platform integration allows you to build custom Messenger Bots using Jovo.'
+---
+
+# Facebook Messenger Platform Integration
+
+The Facebook Messenger [platform integration](https://v4.jovo.tech/docs/platforms) allows you to build custom Messenger bots using Jovo.
+
+
+## Getting Started
+
+You can install the plugin like this:
+
+```sh
+$ npm install @jovotech/platform-facebookmessenger
+```
+
+Add it as plugin to your [app configuration](https://v4.jovo.tech/docs/app-config), e.g. `app.ts`:
+
+```typescript
+import { App } from '@jovotech/framework';
+import { FacebookMessengerPlatform } from '@jovotech/platform-facebookmessenger';
+// ...
+
+const app = new App({
+  plugins: [
+    new FacebookMessengerPlatform(),
+    // ...
+  ],
+});
+```
+
+## Configuration
+
+You can configure the Facebook Messenger platform in the [app configuration](https://v4.jovo.tech/docs/app-config), for example `app.ts`:
+
+```typescript
+import { FacebookMessengerPlatform } from '@jovotech/platform-facebookmessenger';
+
+// ...
+
+const app = new App({
+  plugins: [
+    // For example NLU integration
+  ],
+  sessionExpiresAfterSeconds: 900
+});
+```
+
+Options include:
+
+- `plugins`: Add an [NLU integration](#nlu-integration) here.
+- `sessionExpiresAfterSeconds`: Defines when a request should be treated as a new session. Take a look at [session data](#session-data) for more information.
```

Comment:
During development, I thought it would be cleaner to pass a `session`-object that has the same structure as `session` in `StoredElement`. This way we can also make it easier in the future to configure other parts of the session that is persisted.
So this means that the configuration would rather look like this:
```
new FacebookMessengerPlatform({
  plugins: [
    // For example NLU integration
  ],
  session: {
    expiresAfterSeconds: 900,
  },
});
```
But I am also open to discuss this.

Reply:
Makes sense! Seems like I looked at an old commit